spec/features/01_user_views_index_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+feature "visitor sees a list of cities" do
+  scenario "sees a list of cities and link for new city" do
+    boston = Location.create(name: 'Boston', state: "MA", description: 'Best city in the world')
+    newyork = Location.create(name: 'New York', state: "NY", description: 'The big apple')
+
+    visit locations_path
+
+    expect(page).to have_content "Boston"
```

Comment:
These tests may pass, but they can be stronger and more descriptive. 

Instead of hard-coding the string that matches the value you passed into the model on line 5, referencing the `boston` variable like so:

`expect(page).to have_content boston.state`

That way, someone reading your test has a better ability to understand *what* this is testing (I have an object, I see its 'state' :D)